cluster/ci/monitoring/mixins/grafana_dashboards/configresolver.jsonnet

```diff
@@ -101,20 +142,39 @@ dashboard.new(
 .addPanel(
     (graphPanel.new(
         'Config Resolver Error Rate',
-        description='sum(increase(configresolver_error_rate[4h])) by (error)',
+        description='sum(increase(configresolver_error_rate[${range}])) by (error)',
         datasource='prometheus',
         legend_alignAsTable=true,
         legend_rightSide=true,
         legend_values=true,
         legend_current=true,
     ) + legendConfig)
     .addTarget(prometheus.target(
-        'sum(increase(configresolver_error_rate[4h])) by (error)',
+        'sum(increase(configresolver_error_rate[${range}])) by (error)',
         legendFormat='{{error}}',
     )), gridPos={
     h: 9,
     w: 24,
     x: 0,
     y: 0,
   })
+.addPanel(
+    (graphPanel.new(
+        'Config Resolver Requests Over Time',
+        description='sum(increase(configresolver_http_request_duration_seconds_bucket[${range}]))',
```

Comment:
Why `increase` and not `rate`?

Reply:
It seems like rate doesn't give the actual number of requests (unless maybe I'm using incorrectly?): https://grafana-prow-monitoring-stage.svc.ci.openshift.org/d/1FNwBpTZk/ci-operator-configresolver-dashboard-rate?orgId=1&from=1571163351929&to=1571249751929&var-range=3h

Reply:
I believe rate is per second increase